bob-apps/Cargo.toml

```
@@ -18,6 +18,9 @@ edition = "2021"
 default = ["mimalloc"]
 mimalloc = ["dep:mimalloc"]
 mimalloc-secure = ["dep:mimalloc", "mimalloc/secure"]
+# Important: can't be used with panic = "abort"
+# Please use profile "jemalloc-profile"
+jemalloc-profile = ["jemallocator", "jemallocator/profiling"]
```

Please, use `dep` prefix for dependencies: 
```suggestion
jemalloc-profile = ["dep:jemallocator", "jemallocator/profiling"]
```

---

It doesn't work with features, "jemallocator/profiling" can't be used with "dep:jemallocator"

---

I've fixed mimalloc-secure as it was not compiling as well